src/saving/Save.cs

```diff
@@ -194,32 +202,35 @@ private static void WriteDataToSaveFile(string target, string justInfo, string s
 
         if (info)
         {
-            if (string.IsNullOrEmpty(infoStr))
-            {
-                throw new IOException("couldn't find info content in save");
-            }
-
-            infoResult = ThriveJsonConverter.Instance.DeserializeObject<SaveInformation>(infoStr);
+            infoResult = string.IsNullOrEmpty(data.infoStr) ? SaveInformation.CreateBroken() : ThriveJsonConverter.Instance.DeserializeObject<SaveInformation>(data.infoStr);
         }
 
         if (save)
         {
-            if (string.IsNullOrEmpty(saveStr))
+            if (string.IsNullOrEmpty(data.saveStr))
             {
                 throw new IOException("couldn't find save content in save file");
             }
 
             // This deserializes a huge tree of objects
-            saveResult = ThriveJsonConverter.Instance.DeserializeObject<Save>(saveStr);
+            try
+            {
+                saveResult = ThriveJsonConverter.Instance.DeserializeObject<Save>(data.saveStr);
+            }
+            catch (Exception ex)
+            {
+                GD.PrintErr("Broken save game: " + ex.Message);
```

Comment:
This should have a different error message, something like "Error deserializing save info object: "